index.md

```diff
@@ -72,3 +72,83 @@ Once operational, Helen finds a verification agent, Lynn, who was able to inspec
+
+
+# Focal Use Cases
+
+## Background
+
+As pressure mounts to balance the yearly global carbon budget due to increasing risk to life, property and markets due to climate related.
+
+Carbon accounting, markets and politics. Long lobbied by scientists and citizen groups, corporations and governments are finally taking climate related risk seriously. As voluntary and compliance markets for carbon, corporate science based reporting, emissions taxes and other mechanisms become more common, the demands for carbon accounting integrity become more challenging to meet.
+
+Carbon accounting and science: Carbon accounting should follow the natural carbon cycle. Emissions (respiration) and sequestration (inspiration). The global carbon budget must not simply be brought to net zero as quickly as possible, it must also achieve a carbon negative balance for some time to bring us back to pre-industrial levels if we want to guarantee conditions conducive to biosphere functioning as humans have evolved.
+
+There are three major types of carbon credits.
+
+**Avoided emissions** -- a factory spent money to increase efficiency and can claim credits as they reduce emissions). 
+
+**Renewable Energy Credits** -- a city shifts from fossil fuel based energy to renewable energy and claims credits associated with the quantification of carbon emissions reduction
+
+**Natural carbon or “living carbon”** -- Encapsulated by “terraculture” and “mariculture” (AFOLU: Agriculture, Forestry and other Land Use) and Blue Carbon in which sequestration and/or maintenance of carbon stocks in living systems is monitored and quantified.
+
+## Scenario
+Acme Ag and their ecosystem of farmers, processors, and logistics providers, would like to meet the ambitious goals of carbon neutrality by 2030, and even explore if providing carbon credits to industries that cannot hope to achieve carbon negative operations (such as the energy sector, could become a long term profit center.
+
+Acme Ag wants to be able to have accurate carbon reporting internally, as well as participate in markets to buy and sell carbon credits to ensure year over year they can meet carbon accounting goals 1st and position themselves to be a provider of carbon to the market they believe is coming, in partnership with their farmers.
```

Comment:
"in partnership with their farmers." is hanging a bit.

Perhaps move to start "Acme Ag, in partnership with their farmers, wants ..."